lib/Runtime/Base/CompactCounters.h

```diff
@@ -11,32 +11,40 @@ namespace Js
 {
     template<class T, typename CountT = T::CounterFields>
     struct CompactCounters
-    {
+    {    
+        template<typename CountT>
```

Comment:
Yes, see my reply to Doug's comment. I'll update it but currently it's blocked on an asmjs stack alignment issue.